### PR TITLE
generate example README

### PR DESCRIPTION
**Pull request by Schwusch**

### CODE REVIEW

File: generator/lib/generate_command.dart [hunk 1]
```diff
@@ -155,6 +159,7 @@ class GenerateCommand extends Command {
           pubspecFile.writeAsStringSync(pubspecYaml);
           serviceFile.writeAsStringSync(serviceText);
           readmeFile.writeAsStringSync(buildReadmeMd(api));
+          exampleFile.writeAsStringSync(buildExampleReadme(api));
```
Comment (isoos) [inline generator/lib/generate_command.dart]: Nit: also need formatting.

Comment (Schwusch) [reply generator/lib/generate_command.dart]: Which one, the README or the line in question? I ran `dartfmt` before committing

Comment (isoos) [reply generator/lib/generate_command.dart]: Ah, my bad, I thought that it will be a dart file, not a readme.

Comment (isoos) [reply generator/lib/generate_command.dart]: I mean *not a markdown...

**Pull request merged**